src/Bellhop.js

```diff
@@ -269,11 +269,17 @@ export class Bellhop extends BellhopEventDispatcher {
    *  @param {Boolean} [runOnce=false] If we only want to respond once and then remove the listener
    */
   respond(event, data = {}, runOnce = false) {
-    const internalCallback = e => {
+    const internalCallback = async function(e){
```

Comment:
I love that the diff is so small 👍

Reply:
I smile every time I see or get to use async/await in JS